packages/dd-trace/src/encode/0.4.js

```diff
@@ -271,29 +271,62 @@ class AgentEncoder {
     }
   }
 
+  _encodeMetaStruct (bytes, value) {
+    const entries = Array.isArray(value) ? [] : Object.entries(value)
+    const validEntries = entries.filter(([_, v]) =>
```

Comment:
did you know doing this is valid javascript ?
```suggestion
    const validEntries = entries.filter(([, v]) =>
```

Reply:
I don't have strong opinions, just saying it's possible

Reply:
is valid, but my eyes cry blood when I see a `[, something]`

Reply:
Anyway, I've returned to the previous approach, so this doesn't matter now.